pom.xml

```diff
@@ -41,7 +41,7 @@
     </modules>
 
     <properties>
-        <backbase-bom.version>2021.10</backbase-bom.version>
+        <backbase-bom.version>2022.04</backbase-bom.version>
```

Comment:
2022.04.4 has been released. 2022.04.5 is planned for today.

Reply:
fixed